### ISSUE TITLE
expose base url through up

### ISSUE THREAD

**Issue by jlsjonas** [apex/up#251]
Besides exposing a PORT, exposing the base url for the specific apigateway would be very benefitial; especially since it deploys 3 stages by default.
this would allow url-sensitive projects to work across all stages without complications

**Comment by tj** [comment]
Agreed. I'd love to get rid of the base url in general since it's annoying to develop with, but I'm not 100% sure if AWS is planning to get away from that
